### PR TITLE
add support for port package manager

### PR DESCRIPTION
**Pull request by bhavinthaker**
add support for port package manager along with brew;
show compile-time configs;
print run-time env vars;
print link to MXNet tutorials;
print brew/port and pip package dependencies.

@piiswrong @madjam @sandeep-krishnamurthy @Roshrini

### CODE REVIEW

**Comment by cjolivier01** [comment]
Yes, so libmxnet.so is linked to a different OpenCV shared object than the
OpenCV python module (cv2.so).  There also appears to be a similar conflict
with two numpy versions. Try to remove your personal version of OpenCV and
rebuild.  try to use the official distributions of OpenCV rather than
personally-built ones or ones acquired by other means if you get this sort
of problem (or be sure to remove the official versions to use your own
build, etc).

File: setup-utils/install-mxnet-osx-python.sh [hunk 1]
```diff
@@ -25,37 +25,134 @@ then
...
+		echo "Your macOS version is:"
+		sw_vers
+		echo " "
+		exit 1
+	fi
+} # chk_mac_vers()
+
+install_brew() {
+	echo " "
+	echo "BEGIN: Check/Install/Update Homebrew"
```
Comment (bhavinthaker) [inline setup-utils/install-mxnet-osx-python.sh]: I will make the change. Thanks.

**Comment by bhavinthaker** [comment]
I have modified the script to include user's PATH to have precedence over the script's PATH so that the user's python (for example, anaconda python) will be used. 

Rahul and I ran "pip uninstall opencv-python" and it removed this warning:

objc[99608]: Class CaptureDelegate is implemented in both /usr/local/opt/opencv/lib/libopencv_highgui.2.4.dylib (0x10422dcd0) and /Users/huilgolr/Library/Python/2.7/lib/python/site-packages/cv2/cv2.so (0x10e1e2658). One of the two will be used. Which one is undefined.
RuntimeError: module compiled against API version 0xa but this version of numpy is 0x9
RuntimeError: module compiled against API version 0xa but this version of numpy is 0x9

**Comment by bhavinthaker** [comment]
See: https://github.com/apache/incubator-mxnet/pull/7276

**Pull request closed without merging**